.github/workflows/rocky8.yml

```diff
@@ -7,3 +7,6 @@ jobs:
     uses: ./.github/workflows/task-unit-test.yml
     with:
       container: rockylinux:8
+    defaults:
+      run:
+        shell: bash -l -eo pipefail {0}
```

Comment:
Is this needed? We set it again in `task-unit-test.yml`

Reply:
leftovers